src/main/java/com/minecolonies/coremod/inventory/api/AbstractCombinedItemHandler.java

```diff
@@ -0,0 +1,264 @@
+package com.minecolonies.coremod.inventory.api;
+
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.nbt.NBTTagInt;
+import net.minecraft.nbt.NBTTagList;
+import net.minecraft.util.text.ITextComponent;
+import net.minecraft.util.text.TextComponentString;
+import net.minecraft.world.IWorldNameable;
+import net.minecraftforge.common.util.Constants;
+import net.minecraftforge.common.util.INBTSerializable;
+import net.minecraftforge.items.IItemHandler;
+import net.minecraftforge.items.IItemHandlerModifiable;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Arrays;
+
+/**
+ * Abstract class wrapping around mutli
+ */
+public abstract class AbstractCombinedItemHandler
+    implements IItemHandler, IItemHandlerModifiable, INBTSerializable<NBTTagCompound>, IWorldNameableModifyable
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') "IItemHandler" is already extended by "IItemHandlerModifiable"; there is no need to implement it here. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1939)